modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/GlobalResources/index.js

```diff
@@ -0,0 +1,13 @@
+$(function (){
+    var l = abp.localization.getResource("CmsKit");
+    
+    var service = volo.cmsKit.admin.globalResources.globalResourceAdmin;
+
+    $('#SaveResourcesButton').on('click','',function(){
```

Comment:
If we have a single save button, why we have separate service methods (`setGlobalStyle` and `setGlobalScript`), and we have only one method for get.
I think this is not consistent. Unify these two methods into a single method and make a single AJAX call.